docs/help/support-guidelines.md

```diff
@@ -1,5 +1,86 @@
+- GitHub Discussions
+  - Questions
+  - Unconfirmed bugs
+  - Feature requests
+  - Design discussions
+- GitHub Issues
+  - Confirmed bugs
+- Slack
+  - Instant messaging between contributors
+- ROS Discourse
+  - General topics that should be widely announced
+
+The following sections are detailed descriptions of each resource.
+
+## Documentation sites
+
+[Docs guide](docs-guide.md) shows the list of useful documentation sites.
+Visit them and see if there is any information related to your problem.
+
+Note that the documentation sites aren't always latest and perfect.
+If you find some information is wrong, unclear, or missing in Autoware docs, feel free to submit a pull request following our [contribution guidelines](../contributing/index.md).
+
+!!!warning
+
+    Since this documentation site is still under construction, there are some empty pages.
+
+## GitHub Discussions
+
+If you have a problem with Autoware, check existing issues and questions and search similar issues first.
+
+- [Issues](https://github.com/autowarefoundation/autoware/issues)
+
+  Note that Autoware has multiple repositories listed in [autoware.repos](https://github.com/autowarefoundation/autoware/blob/main/autoware.repos).  
+  It is recommended to search across the repositories.
+
+- [Questions](https://github.com/autowarefoundation/autoware/discussions/categories/q-a)
+
+If no answer was found, create a new question thread [here](https://github.com/autowarefoundation/autoware/discussions/categories/q-a).
+If your question is not answered in a timely manner (about a week), mention maintainers and remind them.
+
+Also, there are other discussion types such as [feature requests](https://github.com/autowarefoundation/autoware/discussions/categories/feature-requests) or [design discussions](https://github.com/autowarefoundation/autoware/discussions/categories/design).
+Feel free to open or join such discussions.
+
+## GitHub Issues
+
+If you have a problem and you have confirmed it is a bug, find the appropriate repository and create a new issue there.
+If you can't determine the appropriate repository, you should not create an issue yet.
+
+!!!warning
+
+    Do not create issues for questions or unconfirmed bugs. If such issues are created, maintainers will transfer them to GitHub Discussions.
+
+If you want to fix the bug by yourself, discuss the approach with maintainers and submit a pull request.
+
+## Slack
+
+Autoware has a Slack workspace for casual communication between contributors.
+
+For example, this is a good place for the following activities:
+
+- Introduce yourself to the community.
+- Chat with contributors.
+- Take a quick straw poll.
+
+Note that it is not the right place to get help for your issues.
+
+## ROS Discourse
+
+If you want to widely discuss a topic with the general Autoware community or ask a question not related to Autoware's bugs, post to [the Autoware category on ROS Discourse](https://discourse.ros.org/c/autoware).
```

Comment:
```suggestion
If you want to widely discuss a topic with the general Autoware and ROS community or ask a question not related to Autoware's bugs, please post to [the Autoware category on ROS Discourse](https://discourse.ros.org/c/autoware).
```

Reply:
I guess we don't use the Autoware category for ROS community.

Reply:
But anyone in the ROS community will be able to see and join to the discussion there.

Reply:
Hmm, I think it should be added here.
https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/help/docs-guide.md

But if you strongly want to add it here, I will apply after you remove `please`.

Reply:
I think you can add it here, removed `please`.

Reply:
Okay!